### ISSUE TITLE
Dropout bug in model.py

### ISSUE THREAD

**Issue by y0un5** [kuanghuei/clean-net#2]
Drop out logic in `model.py` doesn't work.

https://github.com/kuanghuei/clean-net/blob/master/model.py#L123-L136

```python

h0 = tf.contrib.layers.fully_connected(inputs=v_S, ...

if is_training:
    h = tf.nn.dropout(h0, keep_prob=self.keep_prob)

h = tf.contrib.layers.fully_connected(inputs=h0, ...
```

`h = tf.nn.dropout(h0, keep_prob=self.keep_prob)` should be `h0 = tf.nn.dropout(h0, keep_prob=self.keep_prob)`.

**Comment by kuanghuei** [comment]
Thanks for pointing this out. I have committed the fix.
